src/scales/scale.radialLinear.js

```diff
@@ -499,8 +490,9 @@ export default class RadialLinearScale extends LinearScaleBase {
 				);
 			}
 
-			ctx.fillStyle = tickOpts.color;
-			ctx.fillText(tick.label, 0, -offset);
+			renderText(ctx, tick.label, 0, -offset, tickFont, {
```

Comment:
backdrop could be in renderText too. But that can be done later when needed somewhere else

Reply:
Yeah, let's do that after (but I agree that it could be there)